plugins/android/src/main/java/tools/forma/android/feature/AndroidLibrary.kt

```diff
@@ -94,9 +113,7 @@ private fun populateManifest(
             parentFile.mkdirs()
             createNewFile()
             writeText(
-                """<?xml version="1.0" encoding="utf-8"?>
-                           <manifest package="$packageName"/>
-                """.trimIndent()
+                """<manifest package="$packageName"/>"""
```

Comment:
Why we are skipping xml version here?

Reply:
I've never seen any effect of specifying xml version in android xml files like manifest or resources, I can roll back this